src/components/ContinuationIn/ManualBusinessInfo.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+  <div id="manual-business-info">
+    <!-- inactive = ready to click link to start -->
+    <template v-if="!active">
+      <v-row no-gutters>
+        <v-col
+          cols="12"
+          sm="3"
+        >
+          <!-- empty column to line up with ExtraproRegistration label -->
+        </v-col>
+
+        <v-col
+          cols="12"
+          sm="9"
+        >
+          <div class="font-14 ml-3">
+            Not extraprovincially registered in B.C.?
+            &nbsp; &nbsp;
+            <a @click="onClick()">Enter your business information manually</a>
```

Comment:
What do you think of changing this function's name to something more specific Sev? Like `onNotExtraProClick`. Something like that.

Reply:
I only implemented this sub-component to get all the interations worked out. There is another ticket to actually implement this component, and its developer can rename this as they see fit.

This is actually a pretty simple component at the moment, so I think "onClick" is straightforward and not confusing 😁 

I wouldn't label this event handler according to the other sub-component. Best for components to know as little as possible about the world outside of them.

A better method name might be "setActive()". Thoughts?

Reply:
I see, OK Sev. That's fine, I think let's keep as is (`onClick()`). Thanks!